Controller/TwigExceptionController.php

```diff
@@ -44,7 +44,7 @@ public function getTemplating()
     }
 
     /**
-     * {inheritDoc}
+     * {inheritDoc}.
```

Comment:
:+1: @xabbuh fixed it in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1291